conf/config.go

```diff
@@ -373,6 +373,51 @@ func GetProcessingConfiguration(configuration *ConfigStruct) ProcessingConfigura
 	return configuration.Processing
 }
 
+func updateStorageCfgFromClowder(configuration *ConfigStruct) {
+	// get DB configuration from clowder
+	configuration.Storage.PGDBName = clowder.LoadedConfig.Database.Name
+	configuration.Storage.PGHost = clowder.LoadedConfig.Database.Hostname
+	configuration.Storage.PGPort = clowder.LoadedConfig.Database.Port
+	configuration.Storage.PGUsername = clowder.LoadedConfig.Database.Username
+	configuration.Storage.PGPassword = clowder.LoadedConfig.Database.Password
+}
+
+func updateBrokerCfgFromClowder(configuration *ConfigStruct) {
+	// make sure broker(s) are configured in Clowder
+	if len(clowder.LoadedConfig.Kafka.Brokers) > 0 {
+		configuration.Kafka.Addresses = ""
+		for _, broker := range clowder.LoadedConfig.Kafka.Brokers {
+			if broker.Port != nil {
+				configuration.Kafka.Addresses += fmt.Sprintf("%s:%d", broker.Hostname, *broker.Port) + ","
+			} else {
+				configuration.Kafka.Addresses += broker.Hostname + ","
```

Comment:
I'm not sure I understand the config loading, why not use a slice since we [have to do it later](https://github.com/RedHatInsights/ccx-notification-service/pull/704/files#diff-1e1d79894321962be0d7ee31c370d15657f4d85ae4bb31acf1b3394d7d1590ceL54) anyway, instead of this string manipulation?

Reply:
Viper does not parse the slice automatically from an env var, and this trick is way less complicated than having to handle the env var when loading the configuration.